Source/WebKit/UIProcess/Extensions/WebExtensionController.h

```diff
@@ -174,6 +175,8 @@ class WebExtensionController : public API::ObjectImpl<API::Object::Type::WebExte
     // IPC::MessageReceiver
     void didReceiveMessage(IPC::Connection&, IPC::Decoder&) override;
 
+    void createWebExtensionControllerHelper();
```

Comment:
ditto

Reply:
I would say something like `initializePlatform()` or `platformInitialize()` so it can be used for other things that might be platform specific, in this case the Cocoa platform.